### ISSUE TITLE
VPAT — Default Font Color (contrast ratio issue)

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#4460]
As noted in https://www.w3.org/TR/WCAG20-TECHS/F24.html, some users with vision impairment may set their own default text and background colors, within their browser, to help them read sites that do not define these attributes.  Therefore,  both `color` and `background-color` CSS properties should be defined in a way that operate throughout all views.

Submitty does define background colors that operate in all views, but not a text color.  Text colors are defined in CSS, but only for certain situations.  In many places, the reason text is black is because the browser's default font color is black.

Perhaps a vision impaired user set his browser to default of a black background with white text.  Submitty will render white and off-white background colors as intended, but now some text will be white against Submitty's white and off-white backgrounds, therefore becoming unreadable (see attached screenshot of gradeables list for an example).

![Screen Shot 2019-09-11 at 6 51 44 PM](https://user-images.githubusercontent.com/12703624/64741601-27a1b400-d4c7-11e9-80fa-e3b783db8e9c.png)

The goal is to ensure that all text will have a defined color with appropriate contrast ratio.  One suggestion is to define `color` property as black within `body` in CSS.  Through inheritance, all text that did not have a specifically defined color will now be black, even when the browser's default is white.

Following up with this consideration — a challenge is that there are multiple CSS files in use with some redefining properties within the same element.  Notably, `global.css` and `reset.css` attempt to define some of the same properties within `body`.  This can further confuse inheritance which may present difficulties in correcting this issue.  Although my observation is that most views seem to favor inheritance from `global.css`

I also observed that the discussion forum appears to favor inheritance from `reset.css` rather than `global.css`.  In addition, setting `color` to black within `body` has made some text black that may have been intended for another color (such as white or gray).  See screenshot, below (circled areas).

![Screen Shot 2019-09-11 at 7 18 25 PM](https://user-images.githubusercontent.com/12703624/64742173-486b0900-d4c9-11e9-8e48-4227d414696a.png)

**Comment by Dharmesh-Poddar** [comment]
@pbailie can I work on it.

**Comment by elihschiff** [comment]
@Dharmesh-Poddar oops, sorry I should have claimed this issue. I was already working on it, my bad.
